### PR TITLE
few minor improvements

### PR DESCRIPTION
**Pull request by mcparana**
- #957 Add unique key for site_tmplvar_contentvalues
- Removed XHTML markup for self-closing tags in template and chunks
- mm_rules improvement: autogeneration of tabs and moving tvs to tabs by category
- schema.org structured data support in breadcrumbs snippet
- some minor markup changes (empty lines,indents etc)

### CODE REVIEW

**Comment by dmi3yy** [comment]
Thanks but this will be in 1.2.1

**Comment by mcparana** [comment]
ok=)

File: manager/actions/search.static.php [hunk 1]
```diff
@@ -92,7 +92,7 @@
   $searchlongtitle = $modx->db->escape($_REQUEST['searchfields']);
 
   if(isset($_REQUEST['url']) && $_REQUEST['url']!=='') {
-    if(is_int($_REQUEST['url'])) {
+    if(is_int((int)$_REQUEST['url'])) {
```
Comment (Deesen) [inline manager/actions/search.static.php]: Will is_int((int)$xx)) not always return true..?

Comment (mcparana) [inline manager/actions/search.static.php]: maybe (int)$_REQUEST['url']==$_REQUEST['url'] will be correct.

Comment (mcparana) [inline manager/actions/search.static.php]: made commit for that

**Comment by dmi3yy** [comment]
1.2.1 only bugfix
add in 1.2.2

**Pull request closed without merging**